torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -418,7 +418,15 @@ void fixDefaultRNNState(
   Node* unsqueezed_batch_size = graph->create(onnx::Unsqueeze, 1);
   unsqueezed_batch_size->insertBefore(n);
   unsqueezed_batch_size->addInput(batch_size->outputs()[0]);
-  unsqueezed_batch_size->is_(attr::axes, {0});
+  if (opset_version >= OPSET_VERSION_13) {
```

Comment:
nit: Would it be better that we create a new method with a meaningful name to increase readability and let people know why we have these new code? Like, change this line to "if (unsqueezeNeedsMultipleParameters(opset_version))". The method is simple as:

bool unsqueezeNeedsMultipleParameters(int opset_version) {
    return opset_version >= 13;
}

We can put it into helper.h file so this method could be shared among different places, like shape_type_inference.cpp and peehole.cpp.

In addition, for this block of code(line 422 - 428), given they are repeated in different places. Should we also consider create a common method to hold them?

Reply:
Thanks for the suggestion. I'm creating a common helper function for creating onnx unsqueeze node. And adding comments explaining why the difference from opset version.